packages/prisma/schema.prisma

```diff
@@ -31,7 +31,7 @@ model User {
   accounts           Account[]
   sessions           Session[]
   Document           Document[]
-  Subscription       Subscription[]
+  Subscription       Subscription?
```

Comment:
Would this need a migration?

Reply:
No this is just related to code generation, since we already have a unique constraint on `Subscription.userId` there are no other migrations required.